src/catalogue/items/catalogueItemsTable.component.tsx

```diff
@@ -772,6 +772,14 @@ const CatalogueItemsTable = (props: CatalogueItemsTableProps) => {
         </>
       );
     },
+    renderBottomToolbarCustomActions: ({ table }) => (
+      <Typography>
+        {table.getFilteredRowModel().rows.length == catalogueItemsData?.length
+          ? `Total Catalogue Items: ${catalogueItemsData.length}`
+          : `Returned ${table.getFilteredRowModel().rows.length} out of ${catalogueItemsData?.length} Catalogue Items`}
```

Comment:
```suggestion
          : `Displaying ${table.getFilteredRowModel().rows.length} out of ${catalogueItemsData?.length} Catalogue Items`}
```
I think this is more user friendly.

Reply:
The label 'Showing' might be misleading since it implies that the displayed number represents the current viewable catalogue items. However, it actually reflects the filtered total, which may span across multiple pages due to pagination, especially if the count is high

Reply:
Oh yeah sorry I see now it's slightly different to what I thought it was doing. I had assumed it was like the table pagination component, which would say `1-10 of 100` to illustrate that there are more pages to be seen. DataGrid just includes the filtering in the total so if it filtered and had one result it says `1-1 of 1`. DataGateway also does the same, changing the effective total based on the filter. I think "Returned" is alright then in this case.